src/main/java/trade/wayruha/mexc/enums/OrderStatus.java

```diff
@@ -1,9 +1,22 @@
 package trade.wayruha.mexc.enums;
 
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.Arrays;
+
+@Getter
+@AllArgsConstructor
 public enum OrderStatus {
-    NEW,
-    FILLED,
-    PARTIALLY_FILLED,
-    CANCELED,
-    PARTIALLY_CANCELED
+    NEW (1),
```

Comment:
is `aliasNumber` used anywhere in the API? I haven't seen that we get a number instead of a string in the response

Reply:
Yeap, from WS we get numbers instead of normal string.
 see **trade.wayruha.mexc.dto.Order#setOrderTypeByAlias**

Reply:
this dto is (also|primarily) used in REST Get Orders, where they send us a string, e.g., "NEW". Need to make sure OrderStatus we don't break that functionality